manage/users/index.rst

```diff
@@ -33,12 +33,15 @@ Learn more about managing users...
    How? Zammad checks the sender of every incoming message
    at every inbox it monitors,
    and if it doesn’t recognize the address,
-   🪄 **poof**—new customer account!
+   ✨ **poof**—new customer account!
 
    (Your customers never need to set a password.
    Of course, they can if they want to,
    but the account will be there even if they never use it.)
 
+.. hint:: Zammad users are global to the whole instance. 
+   Restriction to specific groups is not possible.
+
```

Comment:
IMO this hint belongs in the groups page, not the users page.

### Why?

Putting this hint here creates more questions than it answers, simply because it's too early:

This is the first page in the entire admin docs—and it's a good first page because everyone everywhere understands the concept of users. The concept of groups, however, is specific to Zammad; it takes some time for new users to create a mental model for this concept.

Thus, it's likely that there are going to be a _lot_ of readers on this page that don't know what a group is yet; there is no way for them to understand the point you are trying to clarify. But there will never be a reader on the group page who doesn't understand what a user is yet.

Reply:
Very good point.
I moved it to groups with commit 42f7a1bf00e99487fca3f04936e60103fc294410